api/src/main/resources/emails/instructionsemail/content.html

```diff
@@ -0,0 +1,51 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+  <style>
+    body {
+      background-color: #F1F1F1;
+      padding-left: 1rem;
+      padding-top: 1rem;
+    }
+    * {
+      font-family: 'Montserrat', sans-serif;
+      font-weight: 300;
+    }
+    .welcome-message {
+      margin-top: 2rem;
+      margin-bottom: 1rem;
+      color: #302C71;
+      font-size: 30px;
+      font-weight: 300;
+      line-height: 36px;
+    }
+    .text {
+      color: rgba(0,0,0,0.87);
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 22px;
+      margin-bottom: 1rem;
+    }
+    .text.thanks {
+      margin-bottom: 0;
+    }
+
+
+  </style>
+</head>
+<body>
+<div><img src="${HEADER_IMG}"></div>
+<div class="welcome-message">Welcome to the <i>All of Us</i> Researcher Workbench!</div>
+<div class="text">
+  Thank you for signing up for the <i>All of Us</i> Researcher Workbench.
+  Your institution expects you to adhere to the following while using the Researcher Workbench.
+</div>
+<div class="text">${INSTRUCTIONS}</div>
+<div class="text">
+  If you have questions, please contact our support team using the Help Desk widget in the
+  Workbench or by emailing support@researchallofus.org.
```

Comment:
I know this is more of Karthik question but should we make this as a link which sends email to support@researchallofus.org just like in welcome email

Reply:
You are right - I'll update